habitat-lab/habitat/datasets/rearrange/samplers/receptacle.py

```diff
@@ -502,9 +506,8 @@ def import_tri_mesh(mesh_file: str) -> List[mn.trade.MeshData]:
 
     :param mesh_file: The input meshes file. NOTE: must contain only triangles.
     """
-    manager = mn.trade.ImporterManager()
-    importer = manager.load_and_instantiate("AnySceneImporter")
-    importer.open_file(mesh_file)
+
+    _importer.open_file(mesh_file)
```

Comment:
Last suggestion (sorry) -- now that you have the importer global as well, it might be good to call `_importer.close()` at the end to free up the memory it used for the last file. Otherwise this is done implicitly only when the instance gets GC'd or when `open_file()` is called again.

You could also keep it local like before and have just the manager global. Compared to plugin manager setup, the actual plugin instantiation is quite lightweight. But both approaches are fine, it's just a matter of preference.

Reply:
I would prefer we did not call close on a global importer if possible, but if the speed gains are substantials, I am okay with it.

Reply:
The importer still exists after it is closed, it should just clear the data cache instead of sticking around after its last use.

Reply:
I moved the importer out of global scope. That should be fine. I think it was just the manager that shouldn't be recreated often.

Reply:
Just to clarify on this -- the `close()` doesn't make the global importer unusable or in some unspecified state. It's just freeing internal memory, which would otherwise be freed on the next call to `open_file()`, which might also happen never.